static/js/global.d.ts

```diff
@@ -8,3 +8,8 @@ declare let zulip_test: any;
 interface JQuery {
     expectOne(): JQuery;
 }
+
+interface Window {
+    last_stacktrace: string;
+    page_params_parse_time: number;
+}
```

Comment:
We should maybe change these globals to module exports and re-export them from `zulip_test`.

Reply:
I will do a follow-up PR for this change. Is that fine?